perfkitbenchmarker/benchmark_spec.py

```diff
@@ -124,8 +140,8 @@ class BenchmarkSpec(object):
   def __init__(self, benchmark_info):
     if (FLAGS.benchmark_config_pair and
         benchmark_info['name'] in FLAGS.benchmark_config_pair.keys()):
-      # TODO(user): Unify naming between config_reader and
-      # perfkitbenchmarker.
+        # TODO(user): Unify naming between config_reader and
```

Comment:
Please avoid reformatting code that's not part of your change, it makes merges and history confusing.